lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -45,3 +45,120 @@ function wp_theme_clean_theme_json_cached_data() {
 		WP_Theme_JSON_Resolver_Gutenberg::clean_cached_data();
 	}
 }
+
+/**
+ * Returns the stylesheet resulting of merging core, theme, and user data.
+ *
+ * @param array $types Types of styles to load. Optional.
+ *                     It accepts 'variables', 'styles', 'presets' as values.
+ *                     If empty, it'll load all for themes with theme.json support
+ *                     and only [ 'variables', 'presets' ] for themes without theme.json support.
+ *
+ * @return string Stylesheet.
+ */
+function gutenberg_get_global_stylesheet( $types = array() ) {
+	/**
+	 * Filters whether the cached global stylesheet can be used.
+	 *
+	 * @since 14.6.0
+	 *
+	 * @param boolean $can_use_cached Whether the cached global stylesheet can be used.
+	 */
+	$can_use_cached = apply_filters(
+		'global_stylesheet_can_use_cache',
```

Comment:
```suggestion
		'wp_get_global_stylesheet_can_use_cache',
```

Let's use the function name (the core one) as the filter name.